src/main/java/com/google/devtools/build/lib/skyframe/toolchains/ToolchainResolutionFunction.java

```diff
@@ -57,6 +57,9 @@
  * selecting the execution platform.
  */
 public class ToolchainResolutionFunction implements SkyFunction {
+  public static final String DEFAULT_PLATFORM_MESSAGE =
+      "For more information on platforms or toolchains see"
```

Comment:
Nit: This could just be the default value of the attribute

Reply:
I actually don't want to do that: I think it's clearer to have the default here (where it will be used) than to indirect it through the rule and attribute.

Reply:
Clearer for Bazel devs yes, but if you make it the attribute default, users see it in docs and know exactly what part of the message they can customize. Since the value is low-effort to maintain anyway, I would thus lean towards making it easy on users.

Reply:
Makes sense, I'm fixing this on import.